evap/staff/importers/json.py

```diff
@@ -212,6 +237,16 @@ def _get_course_type(self, name: str) -> CourseType:
         self.course_type_cache[lookup] = course_type
         return course_type
 
+    def _get_exam_type(self, name: str) -> ExamType:
+        lookup = name.strip().lower()
+        if lookup in self.exam_type_cache:
+            return self.exam_type_cache[lookup]
+
+        # It could happen that the importer needs a new exam type
+        exam_type, __ = ExamType.objects.get_or_create(name_de=name, defaults={"name_en": name})
+        self.exam_type_cache[lookup] = exam_type
```

Comment:
side question: am I missing something, or is it that the cache will not be populated with this lookup the next time unless we also add it as an import name?